src/index.js

```diff
@@ -0,0 +1,324 @@
+const path = require('path');
+const inquirer = require('inquirer');
+
+const Manager = require('./lib/Manager');
+const Engineer = require('./lib/Engineer');
+
+const Intern = require('./lib/Intern');
+const fs = require('fs');
+const generateHTML = (manager, engineers, interns) => {
+  const generateEngineerCard = engineers
+    .map((engineer) => {
+      return `<div class="card" style="width: 18rem">
+  <div class="card-body">
+    <h5 class="card-title">engineer</h5>
+
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">${engineer.id}</li>
+      <li class="list-group-item">email:${engineer.email}</li>
+    </ul>
```

Comment:
You are missing the engineers github url here. Please ensure this is added with as a link.